cfgov/cfgov/urls.py

```diff
@@ -41,6 +40,14 @@
 from v1.views.documents import DocumentServeView
 
 
+if wagtail.VERSION >= (2, 0):
+    from wagtail.contrib.sitemaps.views import sitemap
+    from wagtail.admin import urls as wagtailadmin_urls
+else:
```

Comment:
Any particular reason this isn't a `try`/`except` block?

Reply:
Only lack of coordination. I preferred the more explicit if statement, but the result is the same. We could unify them, but we'll be removing it all on the next iteration.